### PR TITLE
Fix and clean up timeouts for networking tests in large clusters

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref https://github.com/kubernetes/kubernetes/issues/84149

```release-note
NONE
```

### CODE REVIEW

**Comment by wojtek-t** [comment]
@danwinship - seems that something doesn't compile now - please fix

File: test/e2e/framework/service/const.go [hunk 1]
```diff
@@ -56,6 +56,10 @@ const (
 	// LoadBalancerCreateTimeoutLarge is the maximum time to wait for a load balancer to be created/modified.
 	LoadBalancerCreateTimeoutLarge = 2 * time.Hour
 
+	// LoadBalancerPropagationTimeoutDefault is the default time to wait for waiting for pods to
```
Comment (mm4tt) [inline test/e2e/framework/service/const.go]: NIT: Remove ' for waiting ' and extra t in targeted -> ... is the default time to wait for pods to be target(-t)ed

**Review by mm4tt — COMMENTED** [review]
LGTM, just a minor comment comment :)

File: test/e2e/framework/service/const.go (hunk 1 above)
Comment (wojtek-t) [reply test/e2e/framework/service/const.go]: done

**Review by mm4tt — COMMENTED** [review]
/lgtm

**Pull request merged**